groupease-server/src/main/java/io/github/groupease/restendpoint/GroupWebService.java

```diff
@@ -133,77 +133,138 @@ public Group create(@PathParam("channelId") long channelId, @Nonnull GroupCreate
         Member currentUserMember = currentUser.getMemberList().stream()
                 .filter(member -> member.getChannel().getId() == channelId).findFirst().get();
 
-        return groupDao.create(channelId, newGroup.name, newGroup.description, currentUserMember);
+        return groupDao.create(channelId, newGroup.getName(), newGroup.getDescription(), currentUserMember);
     }
 
     /**
      * Updates a {@link Group}
      * @param channelId The ID of the channel the group is part of
      * @param groupId The ID of the group
-     * @param updateGroup The {@link GroupCreateWrapper} object that contains the uploaded JSON wrapped fields
+     * @param updateGroup The changes to make to an existing {@link Group} constructed from the POSTed JSON
      * @return The revised group object
      */
     @PUT
```

Comment:
Your PR description says

> Adds code to existing "update" method to allow a user to leave a group by POSTing the membership list with himself removed.

Glad to see you actually did a `PUT`, which is the correct method for an update.